appengine/pubsub/README.md

```diff
@@ -9,14 +9,16 @@ and [flexible environment](https://cloud.google.com/appengine/docs/flexible/node
 Before you can run or deploy the sample, you will need to do the following:
 
 1. Enable the Cloud Pub/Sub API in the [Google Developers Console](https://console.developers.google.com/project/_/apiui/apiview/pubsub/overview).
-1. Create a topic and subscription.
+1. Create a topic and subscription. The push auth service account must have Service Account Token Creator Role assigned, which can be done in the Cloud Console [IAM & admin](https://console.cloud.google.com/iam-admin/iam) UI. `--push-auth-token-audience` is optional. If set, remember to modify the audience field check in `app.js` (line 112).
```

Comment:
For subscription creation, let's make it clear that it is also GA, so normally someone can just do `gcloud pubsub subscriptions create`. 

But use a second paragraph to note that if they want authenticated push, then follow the instructions and do `gcloud beta pubsub subscriptions create ...` .

Reply:
Done, PTAL!